### ISSUE TITLE
Autolinking shorthands should get line numbers

### ISSUE THREAD

**Issue by tabatkins** [tabatkins/bikeshed#920]
When you use the `-l` flag to get line numbers in your errors, only elements that exist in the original source get those line numbers. If there's a linking error with an autolink shorthand, you don't get a line number, because the actual element got generated by Bikeshed.

I should at least go and infer a "close enough" line number.  I don't do any math on them, so I can write it in as like "Line ~30", if the parent element has `line-number=30`.

**Comment by edent** [comment]
Is this the cause of line numbers not showing up when I run `bikeshed spec -l` on the HTML specification?

I get errors like this - without line numbers.

```
LINK ERROR: Ambiguous for-less link for 'url', please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
Local references:
spec:html-5.3; type:dfn; for:Location; text:url
for-less references:
spec:url; type:dfn; for:/; text:url
<a data-link-type="dfn" data-lt="URL">URL</a>
```
Or am I doing something daft?

**Comment by tabatkins** [comment]
No, that's unrelated. (Check the link the error gives you for how to resolve that error.)

**Comment by edent** [comment]
Ah right. Is there any way to tell *which* autolink needs to be corrected?

**Comment by tabatkins** [comment]
Because there's no line numbers, it's obviously a bit hard. 😀 You'll just have to look for`[=url=]` or something

**Comment by edent** [comment]
I'll just treat it like an adventure puzzle ;-)
